packages/itinerary-body/src/ItineraryBody/index.tsx

```diff
@@ -33,6 +33,7 @@ const ItineraryBody = ({
   setActiveLeg,
   setLegDiagram,
   setViewedTrip,
+  showApproximatePrefixAccessLegs,
```

Comment:
Here is a better prop name for this component and `PlaceRow`: `showApproximateAccessLegTravelTimes`.